src/ui/wxWidgets/Makefile

```diff
@@ -57,7 +59,7 @@ OUTPUTPATH=GCCDebug
 OBJECTPATH=GCCDebug
 BUILDPATHS=$(OBJECTPATH)
 PROGRAM=pwsafe
-LIBS=`$(WX_CONFIG) --debug=yes --unicode=no --libs` -lcore -los -lcore -luuid -lXtst -lX11  -lcurl -lssl -lmagic -l$(YBPERS_LIBS) ${LIBCXX_LD_EXTRA_FLAGS}
+LIBS=`$(WX_CONFIG) --debug=yes --unicode=no --libs std,aui` -lcore -los -lcore -luuid -lXtst -lX11  -lcurl -lssl -lmagic -l$(YBPERS_LIBS) ${LIBCXX_LD_EXTRA_FLAGS}
```

Comment:
Does this change affect every platform? What does it do?

Reply:
Yes, it affects every platform (but shouldn't hurt).

Without this change, it's equivalent to `--libs std`, which lists only standard wx libraries. With `aui` added, wx-config emits wxAui* libraries in addition. pwsafe needs the wxAui libraries.